pkg/speaker/layer2/log.go

```diff
@@ -0,0 +1,61 @@
+package layer2
+
+import (
+	"regexp"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	logRegexpLevel = `\[(?P<level>[A-Z]+)\]`
+	logRegexpMsg   = `(?P<msg>.*)`
+)
+
+func subexps(line []byte) map[string]string {
+	logRegexp := regexp.MustCompile(logRegexpLevel + logRegexpMsg)
```

Comment:
an we make this `logRegexp` a global variable?

Reply:
Both local variables and global variables are acceptable. For memory management purposes, modifying them to global variables might be better, as it can help avoid frequent creation and deletion.